app/components/elements/layout/next_button_component.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
-module Works
-  module Edit
+module Elements
+  module Layout
     # Next button to progress through tabbed form
```

Comment:
These are all just moving elements from `Works::Edit` to `Elements::Layout` for reuse with the collection form.